static/js/components/dialogs/EditVideoFormDialog.js

```diff
@@ -97,28 +182,95 @@ class EditVideoFormDialog extends React.Component {
       noSubmit={false}
       hideDialog={hideDialog}
       onAccept={this.submitForm}
+      onCancel={this.onClose}
       open={open}
+      validateOnClick={true}
     >
-      <div className="mdc-form-field mdc-form-field--align-end">
+      <div className="ovs-form-dialog">
         <Textfield
           label="Title"
           id="video-title"
           onChange={this.setEditVideoTitle}
           value={editVideoForm.title}
+          validationMessage={errors ? errors.title : ''}
         />
         <Textarea
           label="Description"
           id="video-description"
           onChange={this.setEditVideoDesc}
           value={editVideoForm.description}
         />
+        { SETTINGS.FEATURES.ENABLE_VIDEO_PERMISSIONS &&
```

Comment:
This is a lot of code to put in a block like this. Can you move it to a helper method (`renderPermissionOptions` et al) and call that instead if the feature is true?

Reply:
OK